src/gretel_trainer/relational/report/report.py

```diff
@@ -39,120 +53,94 @@ def generated_at(self) -> str:
     def copyright_year(self) -> str:
         return self.now.strftime("%Y")
 
-    @property
-    def relationships(self) -> Relationships:
-        return _table_relationships(self.rel_data)
-
-
-@dataclass
-class Cell:
-    """
-    Dataclass to hold the formatting properties of the cell for rendering Table Relationships table
-
-    Attributes:
-        value (str): The text value of the cell
-        background_color (str, optional): Optional background color, in hexcode format
-        font_weight (str, optional): Optional modifier to font
-    """
-
-    value: str
-    background_color: Optional[str] = None
-    font_weight: Optional[str] = None
+    @cached_property
+    def composite_sqs_score_and_grade(self) -> Tuple[Optional[int], str]:
+        # Add up all the non-None SQS scores and track how many there are.
+        _total_score = 0
+        _num_scores = 0
+        for eval in self.evaluations.values():
+            if eval.individual_sqs is not None:
+                _total_score += eval.individual_sqs
+                _num_scores += 1
+            if eval.cross_table_sqs is not None:
+                _total_score += eval.cross_table_sqs
+                _num_scores += 1
+
+        # Take the average.
+        if _num_scores > 0:
+            score = int(_total_score / _num_scores)
```

Comment:
Since int rounds, can we do the ceiling here instead?

Reply:
Using ceil now.